examples/src/main/scala/examples/hybrid/HLocalInterface.scala

```diff
@@ -64,4 +64,19 @@ class HLocalInterface(override val viewHolderRef: ActorRef,
     posForgerRef ! StopForging
     block = true
   }
-}
\ No newline at end of file
+}
+
+object HLocalInterfaceRef {
+  def props(viewHolderRef: ActorRef,
+            powMinerRef: ActorRef,
+            posForgerRef: ActorRef,
+            minerSettings: HybridMiningSettings): Props = Props(new HLocalInterface(viewHolderRef, powMinerRef, posForgerRef, minerSettings))
```

Comment:
To avoid long lines, you could add a line break after "=".

Reply:
Should I also break long lists of parameters?
Example:
```
 Props(new MyClass(param1, param2, param3, param4, param5, param6, param7, param8))
```
into 
```
 Props(new MyClass(param1, param2, param3, param4, 
                   param5, param6, param7, param8))
```

Reply:
Yes.